README.md

```diff
@@ -344,7 +344,7 @@ module is part of a rarely used feature that can be disabled by unchecking
 ## 3.1 One-time provisioning of shared cloud resources
 
 Most of the cloud resources used by a particular deployment (personal or shared) 
-are provisioned automatically by `make terraform` and `make deploy`. A handful 
+are provisioned automatically by `make deploy` and `make terraform`. A handful 
```

Comment:
I though it would be a good idea to have the sequence of commands be described in the same order. In any case, when `make package` comes along it is already ordered or we can remove it. But I could drop this commit message if you believe we would not gain any value.